editoast/search.yml

```
@@ -20,3 +20,22 @@ operationalpoint:
       ch: opm.data#>>'{extensions,sncf,ch}'
       geographic: ST_AsGeoJSON(ST_Transform(opl.geographic, 4326))::json
       schematic: ST_AsGeoJSON(ST_Transform(opl.schematic, 4326))::json
+
+tracksection:
+  table: osrd_search_tracksection
+  columns:
+    obj_id: string
+    infra_id: integer
+    line_name: string
+    line_code: string
```

```suggestion
    line_code: integer
```


makes `line_code` an integer